src/messages/MOSDPGPush.h

```diff
@@ -43,6 +43,10 @@ class MOSDPGPush : public Message {
     return cost;
   }
 
+  void set_cost(uint64_t c) {
```

Comment:
cost is a public member variable. if we aways access it via its accessors, maybe we can mark it private?